src/event_formatter.rs

```diff
@@ -84,6 +87,30 @@ impl EventFormatter {
             map.serialize_entry("span", &SerializableSpan::new(&span))?;
             // map.serialize_entry("spans", &SerializableContext::new(context))?; TODO: remove
         }
+            //map.serialize_entry("spans", &SerializableContext::new(context))?;
+            let mut trace_id = TraceIdVisitor::new();
+            if trace_id.trace_id.is_none() {
```

Comment:
Is this `if` necessary?

Reply:
You're absolutely right. Thanks for spotting.
I also removed that .expect() from the code